k9mail/src/main/java/com/fsck/k9/mailstore/LocalFolder.java

```diff
@@ -74,6 +75,8 @@
     private static final long serialVersionUID = -1973296520918624767L;
     private static final int MAX_BODY_SIZE_FOR_DATABASE = 16 * 1024;
     static final long INVALID_MESSAGE_PART_ID = -1;
+    private static final int INVALID_UID_VALIDITY = -1;
```

Comment:
Redefining the same constants as in the library. Maybe they should be visible if you need them here?